src/app/models/indicator-params.model.ts

```diff
@@ -0,0 +1,10 @@
+import { ClimateModel } from './climate-model.model';
+import { TimeAggParam } from './time-agg-param.enum';
+
+export interface IndicatorParams {
+    climateModels?: ClimateModel[];
```

Comment:
Just noticed these are all optional. Shouldn't they be required?

Reply:
No, because none of these params are required to make an API request, they all have defaults.